backend/src/routes/user.py

```
@@ -1,37 +1,52 @@
+@jwt_required()
 def create_user():
-    
     data = request.json
-    user = User(username=data['username'], email=data['email'])
+    hashed_password = generate_password_hash(data['password'])
+    user = User(
+        username=data['username'],
+        email=data['email'],
+        password_hash=hashed_password,
+        tenant_id=data['tenant_id'],
+        role=data.get('role', 'user')  # Default role to 'user' if not provided
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)